oauth-implicit/go/main.go

```diff
@@ -0,0 +1,92 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"os/exec"
+	"runtime"
+	"strings"
+	"time"
+)
+
+func main() {
+	openBrowser("http://localhost:5000/")
```

Comment:
You should create variables for the host name and port like with the python tutorial

Reply:
and environment